### ISSUE TITLE
A11y: user focus cannot leave calcite-tree using shit + tab

### ISSUE THREAD

**Issue by joeyHarig** [Esri/calcite-components#2448]
### Summary
Once focused within a `calcite-tree`, the user cannot use shit + tab to focus on elements earlier in the tabindex. 

### Actual Behavior
When focused within a `calcite-tree`, shift + tab will tab backwards in the tab index until the first `calcite-tree-item` of the is reached. Once focused on the first item, shit + tab will set the focus to the selected `calcite-tree-item` or leave the focus set to the first `calcite-tree-item`.

### Expected Behavior
The user should be allowed to escape the `calcite-tree` using shift + tab to navigate to focusable elements earlier in the tabindex.

### Reproduction Steps

1. Go to the `calcite-tree` component in storybook (https://esri.github.io/calcite-components/?path=/story/components-tree--simple)
2. Click on the first tree item
3. Press tab to navigate through the tree
4. Attempt to navigate out of the tree using shift + tab

**Comment by jcfranco** [comment]
Fixed via https://github.com/Esri/calcite-components/issues/683.
